tools/cmake_test.py

```diff
@@ -64,6 +65,51 @@ def _gen_testdir(self):
         return tempfile.mkdtemp(suffix=self._gen_alphanum(),
                                 prefix="%s/test." % self.root)
 
+    def _addr2lines(self, backtrace):
+        if not backtrace:
+            # Maybe we saw a 'Backtrace:' line with no following
+            # address lines, and got an empty list.  Ignore it.
+            return
+
+        # Find our build directory by searching back from binary
+        path_parts = self.binary.split("/")
+        try:
+            vbuild = "/".join(path_parts[0:path_parts.index("vbuild") + 3])
+        except (ValueError, IndexError):
+            sys.stderr.write(
+                f"Could not find vbuild in binary path {self.binary}")
+            return
+        else:
+            addr2line_script = os.path.join(
+                vbuild,
+                "v_deps_build/seastar-prefix/src/seastar/scripts/seastar-addr2line"
```

Comment:
write-line? `\n`

Reply:
Not sure what you mean?

Reply:
sorry i mean to write it on `sys.stderr.write()` i see it has no new line